pkg/testutils/sideeye.go

```diff
@@ -0,0 +1,61 @@
+package testutils
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/DataExMachina-dev/side-eye-go/sideeye"
+	"github.com/stretchr/testify/require"
+)
+
+type testingT interface {
+	require.TestingT
+	TestFatalerLogger
+}
+
+// CaptureSideEyeSnapshot captures a Side-Eye snapshot if the
+// SIDE_EYE_TOKEN env var is set. If the snapshot is captured, the snapshot's
+// URL is logged. Snapshots are captured with a 30s timeout.
+func CaptureSideEyeSnapshot(ctx context.Context, t testingT) {
+	t.Helper()
+
+	if sideEyeToken := os.Getenv("SIDE_EYE_TOKEN"); sideEyeToken == "" {
+		t.Logf("not capturing Side-Eye snapshot; SIDE_EYE_TOKEN env var not set. You can find it in slack or confluence. " +
```

Comment:
Drive-by: I searched for `SIDE_EYE_TOKEN` in Slack and did *not* find the token.

Reply:
Good catch. I'm only hitting `SIDE_EYE_API_TOKEN` in `grep -r SIDE_EYE build/`.

Reply:
do you know a more official place where it's supposed to be that i can refer to? i just search glean for it every time

Reply:
I think what you have is reasonable. I can now find it in Slack searching for `SIDE_EYE_TOKEN` after a message you posted in #_exit-side-eye today.

Reply:
FWIW, the API token for the org can be seen in the app (app.side-eye.io) by logging in (with your @cockroachlabs.com account) and clicking on your avatar image in the top right.

![Screenshot 2024-12-23 at 2 05 49 PM](https://github.com/user-attachments/assets/394c98d7-a3bf-4046-9e6b-67a82b1e1fcf)